ui/pages/confirmations/components/confirm/info/set-approval-for-all-info/set-approval-for-all-info.tsx

```diff
@@ -16,13 +18,27 @@ const SetApprovalForAllInfo = () => {
     selectConfirmationAdvancedDetailsOpen,
   );
 
+  const decodedResponse = useDecodedTransactionData();
+
+  const { value } = decodedResponse;
+
+  const isRevokeSetApprovalForAll =
+    value?.data[0].name === 'setApprovalForAll' &&
+    value?.data[0].params[1].value === false;
```

Comment:
Minor, is it worth some optional chaining here for stability? Could we ever have some of the data without all the params?